netbox_kea/templates/netbox_kea/server_reservation_form.html

```
@@ -52,6 +52,15 @@
         <form method="post" id="reservation-form">
           {% csrf_token %}
           {% render_form form %}
+          {% if subnet_choices %}
+          {# Suggests the server's configured subnets on the Subnet CIDR field. Free typing #}
+          {# stays allowed — a <datalist> only suggests, it never constrains. #}
```

HTMLHint (1.9.2)

[error] Tag must be paired, missing: [ </datalist> ], start tag match failed [ <datalist> ] on line 57.

(tag-pair)